app/views/api/v1/conversations/_conversation.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.conversation_id conversation.id
+json.chat_user conversation.users.where.not(id: current_user.id)
```

Comment:
could we add the `"topic_icon"` in this jbuilder? just to follow the endpoint example from apiary: https://targetapi2021.docs.apiary.io/#reference/0/match-conversation/get-conversations

Reply:
Done!